Small changes (#4)
CTNicholas/liveblocks-docs-assistant: docs/pages/errors/Presence.mdx

  description: "Your 'Presence' type is incorrect and needs to be updated"
---

> ## Why this error occurrs

Author: ````suggestion
## Why this error occurs
````

> You have provided a custom `Presence` types for your application, but the type

Author: ````suggestion
You have provided a custom `Presence` type for your application, but the type
````

you provided isn’t a (completely) valid JSON object. Values like `Date`, `Map`,

> `Set`, functions, classes, or `unknown` aren’t valid JSON. This works prety well.

Author: ````suggestion
`Set`, functions, classes, or `unknown` aren’t valid JSON. This works pretty well.
````

> 
> ## Why this error occurrs
> 
> You have provided a custom `Presence` types for your application, but the type
> you provided isn’t a (completely) valid JSON object. Values like `Date`, `Map`,
> `Set`, functions, classes, or `unknown` aren’t valid JSON. This works prety well.
> 

Author: ````suggestion

## Why this error occurs

You have provided custom `Presence` types for your application, but the types
you provided isn’t a (completely) valid JSON object. Values like `Date`, `Map`,
`Set`, functions, classes, or `unknown` aren’t valid JSON.

````
